SimplePonzi/simplePonzi.qnt

```diff
@@ -48,14 +45,18 @@ module simplePonzi {
   pure def receive(evm: EvmState,
       ponzi: PonziState, investor: Addr, amount: int): Result = {
     if (amount > evm.balances.get(investor)) {
-      error: "Insufficient funds",
-      evm: evm,
-      ponzi: ponzi,
+      {
+        error: "Insufficient funds",
+        evm: evm,
+        ponzi: ponzi,
+      }
      } else if (amount < 11 * ponzi.currentInvestment / 10) {
```

Comment:
I was thinking abstracting the 110% calculation into a helper operator could help readability.